enhancements/cluster-logging/logs-observability-openshift-io-apis.md

```diff
@@ -0,0 +1,602 @@
+* [Journald Node Logs](https://github.com/openshift/cluster-logging-operator/blob/release-5.9/docs/reference/datamodels/viaq/v1.adoc#viaq-data-model-for-journald)
+* [Kubernetes & OpenShift API Events](https://github.com/openshift/cluster-logging-operator/blob/release-5.9/docs/reference/datamodels/viaq/v1.adoc#viaq-data-model-for-kubernetes-api-events)
+
+###### V2
+
+The progression of the ViaQ data model strives to be succinct by removing fields that have been reported by customers as extraneous.
+
+Container log:
+```yaml
+model_version: v2.0
+timestamp:
+hostname:
+severity:
+kubernetes:
+  container_image:
+  container_name:
+  pod_name:
+  namespace_name:
+  namespace_labels:  #map[string]string: underscore, dedotted, deslashed
+  labels:            #map[string]string: underscore, dedotted, deslashed
+  stream:            #enum: stdout,stderr
+message:             #string: optional. only preset when structured is not
+structured:          #map[string]: optional. only present when message is not
+openshift:
+  cluster_id:
+  log_type:          #enum: application, infrastructure, audit
+  log_source:        #journal, ovn, etc
+  sequence:          #int: atomically increasing number during the life of the collector process to be used with the timestamp
+  labels:            #map[string]string: additional labels added to the record defined on a pipeline 
+```
+
+Event Log:
+
+```yaml
+model_version: v2.0
+timestamp:
+hostname:
+event:
+  uid:
+  object_ref_api_group:
+  object_ref_api_version:
+  object_ref_name:
+  object_ref_resource:
+  request_received_timestamp:
+  response_status_code:
+  stage:
+  stage_timestamp:
+  user_groups: []
+  user_name:
+  user_uid:
+  user_agent:
+  verb:
+openshift:
+  cluster_id:
+  log_type:          #audit
+  log_source:        #enum: kube,openshift,ovn,auditd
+  labels:            #map[string]string: additional labels added to the record defined on a pipeline 
+```
+Journald Log:
+
+```yaml
+  model_version: v2.0
+  timestamp:
+  message:
+  hostname:
+  systemd:
+    t:                  #map
+    u:                  #map
+  openshift:
+    cluster_id:
+    log_type:          #infrastructure
+    log_source:        #journald
+    labels:            #map[string]string: additional labels added to the record defined on a pipeline 
+```
+
+### Risks and Mitigations
+
+#### User Experience
+
+The product is no longer offering a "one-click" experience for deploying a full logging stack from collection to storage.  Given we started moving away from this experience when Loki was introduced, this should be low risk.  Many customers already have their own log storage solution so they are only making use of log forwarding.  Additionally, it is intended for the **cluster-observability-operator** to recognize the existance of the internally managed log storage and automatically deploy the view plugin.  This should reduce the burden of administrators.
```

Comment:
The COO should mitigate this risk.  The goal is for the COO to provide the easy out-of-box experience by installing & configuring multiple operators and resources automatically - not just for logging but for other obs. tools. Ideally COO will finally provide a real one-click setup for simple cases, but still allow users to configure the underlying resources if they need greater control.

Reply:
Is there specific verbiage you wish for me to add?